### PR TITLE
Add meeting info and link to draft spec to Readme

### PR DESCRIPTION
**Pull request by bumblefudge**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,12 @@
-# did-pkh-
-We would like to open up the design process for did:pkh to a more open and consultative/deliberative conversation in the open.
+# did-pkh
+
+We would like to open up the design process for the did:pkh [**method specification**](did-pkh-method-draft.md) to a more  consultative/deliberative conversation for the broader W3C-CCG community.
+
+## Meetings
+
+Regular meetings for this CCG work item will be held every two weeks on Thursdays at 1.30PM Eastern Time, on a [workitem-specific jitsi room](meet.w3c-ccg.org/didpkh) hosted by the CCG.  Minutes will be published the same way as other work items.
```
Comment (clehner) [inline README.md:7]:
```suggestion
Regular meetings for this CCG work item will be held every two weeks on Thursdays at 1.30PM Eastern Time, on a [workitem-specific jitsi room](https://meet.w3c-ccg.org/didpkh) hosted by the CCG.  Minutes will be published the same way as other work items.  
```

**Comment by bumblefudge** [comment]
Just to clarify first meeting 2 Dec, not Thanksgiving day :D

**Review by clehner — APPROVED** [review]

**Review by clehner — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1,5 +1,24 @@
...
+
+We would like to open up the design process for the did:pkh [**method specification**](did-pkh-method-draft.md) to a more  consultative/deliberative conversation for the broader W3C-CCG community.
+
+## Meetings
+
+Regular meetings for this CCG work item will be held every two weeks on Thursdays at 1.30PM Eastern Time, on a 
+[workitem-specific jitsi room](https://meet.w3c-ccg.org/didpkh) hosted by the CCG.  Minutes will be published 
+the same way as other work items.  
+
+Our **standing agenda** is first PR review, then issue review (prioritized by labels, then by stalest-first).
```
Comment (TallTed) [inline README.md:11]:
```suggestion
Our **standing agenda** is first [PR](https://github.com/w3c-ccg/did-pkh/pulls) review, then 
[issue](https://github.com/w3c-ccg/did-pkh/issues) review (prioritized by labels, then by 
[stalest-first](https://github.com/w3c-ccg/did-pkh/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-asc)). 
```

File: README.md [hunk 3]
```diff
@@ -1,5 +1,24 @@
-# did-pkh-
-We would like to open up the design process for did:pkh to a more open and consultative/deliberative conversation in the open.
+# did-pkh
+
+We would like to open up the design process for the did:pkh [**method specification**](did-pkh-method-draft.md) to a more  consultative/deliberative conversation for the broader W3C-CCG community.
+
+## Meetings
+
+Regular meetings for this CCG work item will be held every two weeks on Thursdays at 1.30PM Eastern Time, on a
```
Comment (TallTed) [inline README.md:7]:
```suggestion
Regular meetings for this CCG work item will be held every two weeks on Thursdays at [1.30PM Eastern Time](https://www.timeanddate.com/worldclock/), on a 
```

File: README.md [hunk 4]
```diff
@@ -1,5 +1,24 @@
...
+[workitem-specific jitsi room](https://meet.w3c-ccg.org/didpkh) hosted by the CCG.  Minutes will be published 
+the same way as other work items.  
+
+Our **standing agenda** is first PR review, then issue review (prioritized by labels, then by stalest-first). 
+At least one of three work item owners will moderate PR review and issue review (respectively) at each meeting,
+with the prioritization of the latter at the discretion of the chairs. Implementers, researchers, and community
+members are welcome to open issues if they would like clarification of the intended or possible usages of the 
+method. We will use the "next meeting" tag for issues and PRs that we want to discuss soon, so the 
+[next meeting label](https://github.com/w3c-ccg/did-pkh/labels/next%20meeting) view is the fastest way to see
+ n up-to-date agenda for the upcoming meeting(s). We are also glad to take requests for older issues to
```
Comment (TallTed) [inline README.md:17]:
```suggestion
an up-to-date agenda for the upcoming meeting(s). We are also glad to take requests for older issues to 
```

**Review by TallTed — CHANGES_REQUESTED** [review]

**Pull request merged**